### DISCUSSION TITLE
https://mokkapps.de/blog/lazy-load-vue-component-when-it-becomes-visible

### DISCUSSION THREAD

**Discussion by giscus** [Mokkapps/mokkapps-de-discussions#9] [Announcements]
# https://mokkapps.de/blog/lazy-load-vue-component-when-it-becomes-visible

In this blog post, I'll show you how a simple mechanism to lazy load your Vue components if they become visible using the Intersection Observer API.

https://mokkapps.de/blog/lazy-load-vue-component-when-it-becomes-visible

**Comment by smitdesai16** [comment]
We need to replace following line to display loading component.
```js
return h('div', { ref: elRef }, loadingComponent);
```
with
```js
return h('div', { ref: elRef }, loadingComponent.render());
```
